backend/danswer/server/query_and_chat/models.py

```diff
@@ -171,11 +171,32 @@ def check_click_or_search_feedback(cls: BaseModel, values: dict) -> dict:
 
         if click is False and feedback is None:
             raise ValueError("Empty feedback received.")
-
         return values
 
 
 class ChatMessageDetail(BaseModel):
+    message_id: int
```

Comment:
Since so many fields are shared, they should both subclass a parent class to inherit the shared stuff

Reply:
Absolutely